src/libmeasurement_kit/ooni/templates.cpp

```diff
@@ -100,103 +95,8 @@ void dns_query(SharedPtr<Entry> entry, dns::QueryType query_type,
 void http_request(SharedPtr<Entry> entry, Settings settings, http::Headers headers,
                   std::string body, Callback<Error, SharedPtr<http::Response>> cb,
                   SharedPtr<Reactor> reactor, SharedPtr<Logger> logger) {
-
```

Comment:
Implementation moved in `./src/libmeasurement_kit/ooni/templates_impl.hpp` so we can unittest it.